### ISSUE TITLE
Cannot run bundle exec rake redmine:plugins:migrate RAILS_ENV=production NAME=redmine_git_hosting

### ISSUE THREAD

**Issue by aripod** [jbox-web/redmine_git_hosting#673]
I have installed redmine via apt-get and I got it running correctly. Now I am trying to install the plugin by following the [getting started](http://redmine-git-hosting.io/get_started/) but when I have to do `http://redmine-git-hosting.io/get_started/` I get this:

```
/usr/share/redmine$ sudo bundle exec rake redmine:plugins:migrate RAILS_ENV=production NAME=redmine_git_hosting --trace
rake aborted!
NameError: uninitialized constant RSpec
/usr/share/redmine/plugins/redmine_git_hosting/lib/tasks/ci_tools.rake:5:in `block (2 levels) in <top (required)>'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/task_manager.rb:205:in `in_namespace'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/dsl_definition.rb:140:in `namespace'
/usr/share/redmine/plugins/redmine_git_hosting/lib/tasks/ci_tools.rake:3:in `block in <top (required)>'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/task_manager.rb:205:in `in_namespace'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/dsl_definition.rb:140:in `namespace'
/usr/share/redmine/plugins/redmine_git_hosting/lib/tasks/ci_tools.rake:1:in `<top (required)>'
/usr/share/redmine/lib/tasks/redmine.rake:186:in `load'
/usr/share/redmine/lib/tasks/redmine.rake:186:in `block in <top (required)>'
/usr/share/redmine/lib/tasks/redmine.rake:186:in `each'
/usr/share/redmine/lib/tasks/redmine.rake:186:in `<top (required)>'
/var/lib/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/engine.rb:658:in `load'
/var/lib/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/engine.rb:658:in `block in run_tasks_blocks'
/var/lib/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/engine.rb:658:in `each'
/var/lib/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/engine.rb:658:in `run_tasks_blocks'
/var/lib/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/application.rb:452:in `run_tasks_blocks'
/var/lib/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/engine.rb:453:in `load_tasks'
/var/lib/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/railtie.rb:194:in `public_send'
/var/lib/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/railtie.rb:194:in `method_missing'
/usr/share/redmine/Rakefile:6:in `<top (required)>'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/rake_module.rb:28:in `load'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/rake_module.rb:28:in `load_rakefile'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:687:in `raw_load_rakefile'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:96:in `block in load_rakefile'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:178:in `standard_exception_handling'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:95:in `load_rakefile'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:79:in `block in run'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:178:in `standard_exception_handling'
/var/lib/gems/2.3.0/gems/rake-12.0.0/lib/rake/application.rb:77:in `run'
/var/lib/gems/2.3.0/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
/usr/local/bin/rake:23:in `load'
/usr/local/bin/rake:23:in `<main>'
```

Could this be something due to permissions as I am doing everything in /usr/share/redmine and I had to install in (apt-get) and then clone the repositories with sudo as well?
By the way, I am installing it with my user but I know the root password. I did not create a "redmine" user with disabled-password as I have seen on some installation guide. Could this be?

Thanks for the help.

**Comment by kavaliro** [comment]
Same.

**Comment by aripod** [comment]
I did a fresh install of redmine (with bitnami) and when I want to follow the getting started guide, the website is down. [http://www.redmine-git-hosting.io/get_started/](http://www.redmine-git-hosting.io/get_started/)

**Comment by aripod** [comment]
Now that I got access again to the get_started guide, I can follow the steps.
I am stuck at step 2. I have already make the modifications needed as I installed redmine with bitnami and I have cloned and did `git checkout v1.2.x`. Now when I have to do `bundle install --without development test` I get that `Could not locate Gemifle` in my $REDMINE_ROOT directory. There is no Gemlock file there, so in which directory should I do this?

Thanks for the help.

**Comment by aripod** [comment]
Ok, I managed to get to the get a clean install with bitnami, solved all potential issues and checkdout v1.2.x. When I have to do the last step in step 2 from the get_started guide `bundle exec rake redmine:plugins:migrate RAILS_ENV=production NAME=redmine_git_hosting` I get `Plugin redmine_git_hosting was not found`

**Comment by aripod** [comment]
Solved the issue. It was a problem related to the database with bitnami.
